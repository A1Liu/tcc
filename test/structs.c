struct FileId {
  int ident;
  int length;
  int cursor;
};

int ident_of(struct FileId file) { return file.ident; }

int main(int argc, char **argv) {
  struct FileId file;
  int id = ident_of(file);
}
