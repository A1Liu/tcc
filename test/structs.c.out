12
sizeof FileId is 12
sizeof file3.ident is 4
13
